extra/generate-display-names/generate.c

```diff
@@ -36,7 +36,7 @@ generateDisplayName(const char *table) {
 	language = displayLanguage(locale);
 	n += sprintf(n, "%s", language);
 	q += sprintf(q, "locale:%s", locale);
-	free(locale);
+	//free(locale);
```

Comment:
Why are you disabling all the memory freeing?

Reply:
There is a memory issue somewhere but I can't find it (spent considerable time on it). Since the only purpose of this code is to be used at build time, and primarily for Travis, I think it's more important that the code gets executed than that it is bug free.